### PR TITLE
created guide for crimping RJ-45 cables

### PR DESCRIPTION
**Pull request by Ben-Graham**

### CODE REVIEW

File: Cable-Crimping/README.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+-Gree/white stripe
+-Blue
+-Blue/white stripe
+-Green
+-Brown/white stripe
+-Brown
+
+Trim the untwisted wires to a good length.  The insulation of the cable should be just inside the bottom of the RJ-45 connector. Note to orient the cables so that the Orange/white striped wire is on the right when looking at the plastic release latch on the top.
+
+If the wires must be trimmed, trim them in small incrememnts.  Check frequently to ensure that they fit correctly.
```
Comment (bill-luu) [inline Cable-Crimping/README.adoc]: Do you have any tips for someone to help them check that it's in right?

File: Cable-Crimping/README.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+== Connecting the wires to the  RJ-45 connector
+
+Insert the wires into the RJ-45 connector.  Take care to make sure that each wire goes all the way into its appropriate channel, otherwise after crimping it won't work.
+
+== Crimping
+
+Place the connector into the middle hole of the crimp tool and firmly squeeze the handles firmly together to crimp the cable.  You may want to do this more than once to ensure that the cable is properly connected.
```
Comment (bill-luu) [inline Cable-Crimping/README.adoc]: You should specify that the middle hole is designated `8P`

File: Cable-Crimping/README.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
+= How to Crimp Rj45 cables
+
+== Strip the wire
+Make a shallow cut with a knife and run it around the cable 2.5 to 5 cm away from the end of the cable.  The jacket should slide off easily.
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: Asciidoc best practices, one sentence one line. New lines don't do anything in asciidoc.

File: Cable-Crimping/README.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+-Blue
+
+-Blue/white stripe
+
+-Green
+
+-Brown/white stripe
+
+-Brown
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: Asciidoc syntax for bullet points is 
```
* first item
* second item
* third item
```

* first item
* second item
* third item

File: Cable-Crimping/README.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+
+Trim the untwisted wires to a good length.  The insulation of the cable should be just inside the bottom of the RJ-45 connector. Note to orient the cables so that the Orange/white striped wire is on the right when looking at the plastic release latch on the top.
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: One sentence one line.

File: Cable-Crimping/README.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
+= How to Crimp Rj45 cables
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: Capital J

**Comment by enochtsang** [comment]
You can use https://asciidoclive.com to quickly check if your syntax is correct. You have some stuff not working as intended.

File: Cable-Crimping/README.adoc [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+* Blue/white stripe
+* Green
+* Brown/white stripe
+* Brown
+
+Trim the untwisted wires to a good length.
+The insulation of the cable should be just inside the bottom of the RJ-45 connector.
+Note to orient the cables so that the Orange/white striped wire is on the right when looking at the plastic release latch on the top.
+
+If the wires must be trimmed, trim them in small incrememnts.
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: increments typo

File: Cable-Crimping/README.adoc [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+There will be 4 pairs of twisted wires that should now be exposed:
+
+* Solid orange and half orange, half white
+* Solid green and half green, half white
+* Solid blue and half blue, half white
+* Solid brown and half brown, half white.
+
+== Removing the core
+Folding back each of the wires will expose a core of the cable.
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: should it be `the core of the cable`

File: Cable-Crimping/README.adoc [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+* Green/white stripe
+* Blue
+* Blue/white stripe
+* Green
+* Brown/white stripe
+* Brown
+
+Trim the untwisted wires to a good length.
+The insulation of the cable should be just inside the bottom of the RJ-45 connector.
+Note to orient the cables so that the Orange/white striped wire is on the right when looking at the plastic release latch on the top.
```
Comment (enochtsang) [inline Cable-Crimping/README.adoc]: Use the note syntax.

`NOTE: To orient the cables...`

This is actual separate adoc syntax, not just a style thing.

**Comment by enochtsang** [comment]
Bill has merge rights, LGTM. Please test it yourself first Bill.

File: Cable-Crimping/README.adoc [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+== Crimping
+
+Place the connector into the middle hole, designated 8P, of the crimp tool and firmly squeeze the handles firmly together to crimp the cable.
+You may want to do this more than once to ensure that the cable is properly connected.
+
+NOTE: There should be a satisfying click sound.
+A crunching sound means that the cable did not crimp correctly.  
+
+The RJ-45 connector should now be properly working.
```
Comment (bill-luu) [inline Cable-Crimping/README.adoc]: How about `The RJ-45 cable should now be properly working.`

File: Cable-Crimping/README.adoc [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+* Blue
+* Blue/white stripe
+* Green
+* Brown/white stripe
+* Brown
+
+Trim the untwisted wires to a good length.
+The insulation of the cable should be just inside the bottom of the RJ-45 connector.
+
+NOTE: to orient the cables so that the Orange/white striped wire is on the right when looking at the plastic release latch on the top.
```
Comment (bill-luu) [inline Cable-Crimping/README.adoc]: Should be `Note: Orient the cables so that....`

**Pull request merged**